dcp_prototype/frontend/src/components/footer.js

```diff
@@ -8,31 +8,46 @@ const Header = ({ siteTitle }) => {
     <Box
```

Comment:
The footer looks good as is with more data (sticks to the bottom with not too much space). Can we leave the margins?